extensions/wikia/AdEngine/js/lookup/services.js

```diff
@@ -32,24 +31,13 @@ define('ext.wikia.adEngine.lookup.services', [
 			}
 		},
 		bidMarker = ['x', 'x', 'x', 'x', 'x'],
-		realSlotPrices = {},
-		slotAliases = {
-			mercury: {
-				BOTTOM_LEADERBOARD: 'MOBILE_PREFOOTER',
```

Comment:
What about all prebid adapters? We should change MOBILE_PREFOOTER to BOTTOM_LEADERBOARD there. Is it safe to do now?

Reply:
I gonna change `MOBILE_PREFOOTER` to `BOTTOM_LEADERBOARD` in all Prebid.js adapters and ask Nef if that is gonna be fine.